phoenix-core/src/main/java/org/apache/phoenix/exception/SQLExceptionCode.java

```diff
@@ -480,8 +480,11 @@ public SQLException newException(SQLExceptionInfo info) {
                     "table with ROW_TIMESTAMP column."),
     CANNOT_UPSERT_WITH_SCN_FOR_MUTABLE_TABLE_WITH_INDEXES(903,"43M14",
             "Cannot use a connection with SCN set to " +
-                    "upsert data for a mutable table with indexes.");
+                    "upsert data for a mutable table with indexes."),
 
+    UNIMPLEMENTED_CASCADE_FEATURE(904,"43M15","This feature is WIP. Please refer to PHOENIX-5261"),
```

Comment:
Nit: Please leave spaces between different args like other usages above in this file.